docs/content/preview/reference/configuration/yugabyted.md

```diff
@@ -1002,8 +997,9 @@ For example, to set up xCluster replication between two clusters, run the follow
 --replication_id *xcluster-replication-id*
 : The replication ID of the xCluster replication to be set up.
 
---bootstrap_done *xcluster-bootstrap-done*
+--bootstrap_done
 : Indicates that you have completed [bootstrapping](#bootstrap-databases-for-xcluster) of databases.
+: Using this flag means the complete DB schema, objects and data (if any) has been copied from source to target. Any changes made to source after `yugabyted xcluster set_up` command will be reflected on the target.
```

Comment:
: Using this flag indicates that the complete database schema, objects, and data (if any) have been copied from source to target. Any changes made to source after `yugabyted xcluster add_to_replication` command will be reflected on the target.